pom.xml

```diff
@@ -69,7 +86,7 @@
     <dependency>
       <groupId>org.projectlombok</groupId>
       <artifactId>lombok</artifactId>
-      <scope>provided</scope>
+      <optional>true</optional>
```

Comment:
in `vip-report` and `vip-decision-tree` we use `<scope>provided</scope>`. why change to `optional` here?